### ISSUE TITLE
django-image-cropping not working in Django 2.x

### ISSUE THREAD

**Issue by Seykotron** [oscarmlage/django-cruds-adminlte#117]
Hi,

I'm getting crazy with this, using the same code as the Demo, and the frontend part of cropping not working.

There is no error in Inspect Elements, the jQuery is loading fine etc, I dont know what it can be, any guesses?

Thank you!!!
![Captura de pantalla 2019-08-18 a las 15 25 47](https://user-images.githubusercontent.com/2643046/63225026-89267980-c1cc-11e9-90e0-77fd4ba7f990.png)

**Comment by oscarmlage** [comment]
Yeah @Seykotron you're right. Right now the `django-image-cropping` package is not working 😢.

We need to work in the last version integration. Had it tested and running with `Django 1.x` but it crashed with `2.x`. I saw `django-image-cropping` has now [support for Django 2.1](https://github.com/jonasundderwolf/django-image-cropping/commit/37ee678afad79d49c85e976d8168aca6795c58e7) so the code needs to be updated (probably in the next release). Thanks for noticing.

**Comment by Seykotron** [comment]
Let me know if I can give you a hand to fix it, I need this feature hehehe

I also have some modifications in the code for example let __icontains in filter_list, I think it can be very usefull :)

**Comment by oscarmlage** [comment]
Hey @Seykotron, cropping is working here in Demo project with `Django==2.2.3` + `django-image-cropping==1.2.0`

![image](https://user-images.githubusercontent.com/98542/63848878-07cfa380-c991-11e9-80ad-d814e4c0e967.png)

Please give a look to your environment versions and let me  know if it works there. Also if it doesn't please try to prepare something like a demo-project or some bundle that I could download and run here in my environments.

**Comment by oscarmlage** [comment]
I was fifty-fifty wrong in last comment. It is working if you edit/update a record that already has the image, but it is not working if you're adding a new record.
